### PR TITLE
MB-7465 Create a ContactInfoDisplay component to be used on the new Profile page

### PR DESCRIPTION
**Pull request by felipe-lee**
## Description

Create a `ContactInfo` component that can be used on the new Profile page.

## Reviewer Notes

A few questions:
1. Is the placement in storybook ok?
2. Does the CSS look ok. It's somewhat based on the CSS in `src/components/Customer/Review/ShipmentCard/ShipmentCard.module.scss` and `src/components/Customer/Review/Review.module.scss`

## Setup

1. Run storybook:

    ```sh
    make storybook
    ```

2. Navigate to [Customer Components / Profile / ContactInfo](http://localhost:6006/?path=/story/customer-components-profile-contactinfo--default-state)

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-7465) for this change

#### Without Alt Phone
Screen Shot 2021-04-08 at 16 57 59

#### With Alt Phone
Screen Shot 2021-04-08 at 16 58 50

#### Without Alt Phone
Screen Shot 2021-04-08 at 16 59 49

#### With Alt Phone
Screen Shot 2021-04-08 at 17 00 02

### CODE REVIEW

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+ContactInfo.propTypes = {
+  telephone: PropTypes.string.isRequired,
+  secondaryTelephone: PropTypes.string,
+  personalEmail: PropTypes.string.isRequired,
+  phoneIsPreferred: PropTypes.string.isRequired,
+  emailIsPreferred: PropTypes.string.isRequired,
+  residentialAddress: ResidentialAddressShape.isRequired,
+  backupMailingAddress: ResidentialAddressShape.isRequired,
+  backupContact: PropTypes.shape({
```
Comment (YanZ777) [inline src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:124]: Looks like there's a backup contact shape that you can use in `src/types/customerShapes.js`

File: src/components/Customer/Profile/ContactInfo/ContactInfo.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import ContactInfo from 'components/Customer/Profile/ContactInfo/ContactInfo';
+
+export default {
+  title: 'Customer Components / Profile / ContactInfo',
```
Comment (YanZ777) [inline src/components/Customer/Profile/ContactInfo/ContactInfo.stories.jsx:6]: I think we are trying to un nest components, so I'd take out Profile, but I would check with other people.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 1 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:124]: oh nice, thanks!

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, GridContainer, Grid } from '@trussworks/react-uswds';
+
+import contactInfoStyles from './ContactInfo.module.scss';
+
+import { ResidentialAddressShape } from 'types/address';
+import { BackupContactShape } from 'types/customerShapes';
+
+const ContactInfo = ({
```
Comment (YanZ777) [inline src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:10]: Maybe we should rename this component? 
There's an existing ContactInfo component made for onboarding the customer. 
`src/pages/MyMove/Profile/ContactInfo.jsx`

Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:10]: Yeah, that'd probably be good. `ContactInfoList`? `ContactInfoDisplay`? I'm bad at naming haha.

Comment (kimallen) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:10]: I like `ContactInfoDisplay`

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+            <dt>Phone</dt>
+            <dd>{backupContact.telephone}</dd>
+          </dl>
+        </Grid>
+      </Grid>
+    </GridContainer>
+  );
+};
+
+ContactInfo.propTypes = {
```
Comment (kimallen) [inline src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:97]: I don't know what is preferred, but I wonder if pulling in `serviceMember` would be simpler here and then spreading these things in props.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+  telephone: PropTypes.string.isRequired,
+  secondaryTelephone: PropTypes.string,
+  personalEmail: PropTypes.string.isRequired,
+  phoneIsPreferred: PropTypes.bool.isRequired,
```
Comment (YanZ777) [inline src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:101]: I think because either the `phoneIsPreferred` or `emailIsPreferred` could be checked, these should be marked as optional and have default values of false.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 4 above)
Comment (YanZ777) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:97]: My vote is to keep it spread in props since not everything in service member is needed in this component. (I think that would keep it generic)

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 3 above)
Comment (YanZ777) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:10]: I am also bad at naming, but `ContactInfoDisplay` sounds good to me as well.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+            onClick={onEditClick}
+          >
+            Edit
+          </Button>
+        </Grid>
+      </Grid>
+
+      <Grid row>
+        <Grid col className={contactInfoStyles['contact-info-section']}>
+          <dl>
```
Comment (kimallen) [inline src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:48]: If we've decided `dl` is the preferred method of creating this table, I'm wondering if we should replace the `tables` that we are using in the equivalent components for consistency (ex. `ServiceInfoTable`, `PPMShipmentSummary`, `OrdersTable`.  Not that this should necessarily happen in this PR, but at least we should write a story to get things aligned.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 5 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:101]: It raised an error when I had that originally, which is why I changed it to this.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 3 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:10]: K, I'll rename it

**Review by kimallen — COMMENTED** [review]
Looks good to me so far!
There's some visual inconsistencies between the mobile version of this ContactInfo component and the `ServiceInfoTable` component.  I didn't find specs, so not sure which is more accurate to what we want.  Since those two tables will be displayed on the same screen, we should make sure they match.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+  return (
+    <GridContainer className={contactInfoStyles['contact-info-container']}>
+      <Grid row>
+        <Grid col className={contactInfoStyles['contact-info-header']}>
+          <h2>Contact info</h2>
+          <Button
+            unstyled
+            className={contactInfoStyles['edit-btn']}
+            data-testid="edit-contact-info"
+            onClick={onEditClick}
```
Comment (YanZ777) [inline src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:39]: I think there should be a path passed to the onEditClick function, which for now will be the edit profile path (`/moves/review/edit-profile`).

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 5 above)
Comment (kimallen) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:101]: The initial value of those fields is nil, so if they don't click one of them, it will be nil.  If they click one and then unclick, then it becomes false.  What was the error?  Did it ask for a default value?

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 3 above)
Comment (kimallen) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:10]: Actually, to be in line with the other similar components, maybe it should be `ContactInfoTable` (even though you are not semantically using a table)

Comment (kimallen) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:10]: Or `CustomerContactInfoTable`

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx [hunk 8]
```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, GridContainer, Grid } from '@trussworks/react-uswds';
+
+import contactInfoStyles from './ContactInfo.module.scss';
```
Comment (kimallen) [inline src/components/Customer/Profile/ContactInfo/ContactInfo.jsx:5]: Since you are only importing one stylesheet, I'd simplify and just call this `styles`

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 3 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx]: Well should we name this one like those, or as you pointed out in another comment, if we refactor those, we can also rename them.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 5 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx]: Hmm I don't remember because that was one of the earlier errors haha. Let me change it and see.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 6 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx]: Yeah I feel like it'd be good to eventually not use tables for non-tabular data.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 7 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx]: I wasn't sure if it was preferred for the page component that uses this to define that. Like it can define the `onEditClick` function to push to the appropriate url. I can update it to be in this one though if that's preferred.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 8 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx]: ah ok cool, that'll make renaming easier too haha

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss [hunk 9]
```diff
@@ -0,0 +1,56 @@
+@import 'shared/styles/_variables';
+@import 'shared/styles/_basics';
+@import 'shared/styles/colors';
+
+.contact-info-container {
```
Comment (kimallen) [inline src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:5]: I think we are trying to use camelCase in our styles, but maybe @suzubara can confirm

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+    preferredContactMethod = 'Email';
+  }
+
+  return (
+    <GridContainer className={styles['contact-info-container']}>
+      <Grid row>
+        <Grid col className={styles['contact-info-header']}>
+          <h2>Contact info</h2>
+          <Button unstyled className={styles['edit-btn']} data-testid="edit-contact-info" onClick={onEditClick}>
+            Edit
```
Comment (kimallen) [inline src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx:36]: When I select Edit in the mobile view, the blue highlight box seems particularly big.  Not sure if that is desired or not- something for @cosburn to check out

Comment (YanZ777) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx:36]: That seems to be the case for other edit buttons as well from what I can tell. Maybe something should be written up to address all of them at once?

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss (hunk 9 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:5]: Ah yeah, that'd be good to know, I saw both and wasn't sure which was preferred.

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 5 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx]: Huh, nvm, it let me. Must have been something else and I misunderstood the error it gave me and just so happened to have fixed it when I changed it. Idk haha, but regardless, they are now optional and have defaults of false!

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx (hunk 10 above)
Comment (YanZ777) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx:36]: Ok nevermind, I can't find other examples.

Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx:36]: Hmm seems others get big in different ways haha:
https://storybook.dp3.us/?path=/story/customer-components-shipmentcard--ntsr-shipment

Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx:36]: For reference:

This pr:
Screen Shot 2021-04-08 at 19 31 03

Existing one: 
Screen Shot 2021-04-08 at 19 31 14

Comment (YanZ777) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx:36]: Ah! Yes, that's where I saw it. It was on the NTSR shipment card.

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss (hunk 9 above)
Comment (suzubara) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:5]: Yeah camelCase is preferred in CSS modules since you need to be able to refer to the class names as keys in JS, and you won't have to interpolate if they're camelCase, ie
```
// annoying
styles['contact-info-container']

// not annoying
styles.contactInfoContainer
```

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+        clear: both;
+      }
+
+      dd {
+        @include u-margin(0);
+        @include u-width(67%);
+        @include u-float(right);
+        clear: right;
+      }
+    }
```
Comment (suzubara) [inline src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:54]: This is not a blocker _but_ I did want to point out one of the existing components that uses `<dl>` in a similar style - [Shipment Card](https://storybook.dp3.us/?path=/story/customer-components-shipmentcard--hhg-shipment) ([CSS is here](https://github.com/transcom/mymove/blob/master/src/components/Customer/Review/ShipmentCard/ShipmentCard.module.scss), see especially the styling for `dt, dd, .row`) - which uses display: flex instead of floats and will probably be easier to deal with, if we end up rebuilding a lot of the tables with `dl`s. One of the nice things about `dl` is that unlike `ol, ul` it is less strict about child elements, and you can nest other elements like `div`s inside in order to help structure the `dt, dd` elements. IMO it's up to you if you want to switch over to this pattern as part of this PR, but I thought I would mention it.

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss [hunk 12]
```diff
@@ -0,0 +1,56 @@
+@import 'shared/styles/_variables';
+@import 'shared/styles/_basics';
+@import 'shared/styles/colors';
+
+.contact-info-container {
+  @include u-margin-top(3);
+
+  .contact-info-header {
+    h2 {
+      @include u-width(80%);
```
Comment (suzubara) [inline src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:10]: What is the purpose of setting a width on the `h2`? Since the parent element has `display: flex` this shouldn't be necessary

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.stories.jsx [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+
+import ContactInfoDisplay from './ContactInfoDisplay';
+
+import SectionWrapper from 'components/Customer/SectionWrapper';
+
+export default {
+  title: 'Customer Components / Profile / ContactInfoDisplay',
+  component: ContactInfoDisplay,
+  argTypes: {
+    onEditClick: 'go to edit page',
```
Comment (suzubara) [inline src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.stories.jsx:11]: I think this has to be defined as an action in order to hook up to the actions pane:
```
onEditClick: { action: 'go to edit page' },
```

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+  } else if (phoneIsPreferred) {
+    preferredContactMethod = 'Phone';
+  } else if (emailIsPreferred) {
+    preferredContactMethod = 'Email';
+  }
+
+  return (
+    <GridContainer className={styles['contact-info-container']}>
+      <Grid row>
+        <Grid col className={styles['contact-info-header']}>
```
Comment (suzubara) [inline src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx:33]: I don't think we should be using `GridContainer` or `Grid` in this component. `GridContainer` is what applies page-level grid layout (ie, put the whole page content inside a specific box and center it) and `Grid` components are used to align multiple boxes or "columns" in a side-by-side layout. This component consists of a single box (or "column"), the contents of which should be subject to its parent styling. Since this will be rendered inside of a `<SectionWrapper>` you can leave it to that component to contain this one into a specific area & apply margins/padding around it.

In other words, I'd recommend replacing the outer wrapper of this component with:

```
<div className={styles.contactInfoContainer}>
  <div className={styles.contactInfoHeader}>
    <h2>Contact info</h2>
   // ...the rest of the component
</div>
```

Comment (suzubara) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx:33]: alternatively, you could include `SectionWrapper` as part of this component if that's how it will always be displayed. You may not need `styles.contactInfoContainer` since SectionWrapper will apply its own consistent padding. ie:
```
<SectionWrapper className={styles.contactInfoContainer}>
  <div className={styles.contactInfoHeader}>
    <h2>Contact info</h2>
    // ...the rest of the component
</SectionWrapper>
```

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss [hunk 15]
```diff
@@ -0,0 +1,56 @@
+@import 'shared/styles/_variables';
+@import 'shared/styles/_basics';
+@import 'shared/styles/colors';
+
+.contact-info-container {
+  @include u-margin-top(3);
```
Comment (suzubara) [inline src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:6]: The spacing around this component should be handled by its parent (in this case, `SectionWrapper`) - see comments above about the outer markup

**Review by suzubara — COMMENTED** [review]
I left some comments but my only blocking one is about the use of GridContainer, which I don't think should be used here

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss (hunk 12 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:10]: the header was getting squished for some reason. So like "Contact info" was coming out stacked.

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.stories.jsx (hunk 13 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.stories.jsx:11]: ah ok, i'll update that.

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx (hunk 14 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx:33]: I _think_ it'll likely always have the `SectionWrapper`, but just to allow more flexibility, I'll do the `div` based one.

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss (hunk 15 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:6]: K, I'd mainly copied this from one of the existing CSS files. I'll take it out! Less CSS sounds good to me haha.

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss (hunk 11 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:54]: Yeah I hadn't been sure about using divs like that, but if you think that's fine, then I can re-structure it. I don't particularly have a strong opinion and it might let me re-use more of the CSS. Should the CSS be pulled out into a more generic file, or is it fine to just import it as is?

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+          <h2>Contact info</h2>
+          <Button unstyled className={styles['edit-btn']} data-testid="edit-contact-info" onClick={onEditClick}>
```
Comment (suzubara) [inline src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx:35]: i noticed this in another PR & wanted to point it out here: https://github.com/transcom/mymove/pull/6362#discussion_r610740689

> this is a Button component but should probably be a Link (because all it does is navigate to another page)

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss (hunk 11 above)
Comment (suzubara) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:54]: I think this is a great time for a generic file!

File: src/components/Customer/Profile/ContactInfo/ContactInfo.jsx (hunk 7 above)
Comment (YanZ777) [reply src/components/Customer/Profile/ContactInfo/ContactInfo.jsx]: Confirmed with @suzubara, the page component should be defining this, so what you have right now is good!

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss (hunk 12 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:10]: Like this: Screen Shot 2021-04-09 at 14 35 44

Comment (suzubara) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.module.scss:8]: ooh I see. You should be able to give it `flex-shrink: 0;` instead, which will tell the `h2` not to "shrink" when the flex container gets smaller

File: src/testUtils.jsx [hunk 17]
```diff
@@ -3,6 +3,8 @@ import { node, shape, arrayOf, func, string } from 'prop-types';
 import { Provider } from 'react-redux';
 import { createMemoryHistory } from 'history';
 import { ConnectedRouter } from 'connected-react-router';
+import { Router } from 'react-router-dom';
+import { render } from '@testing-library/react'; // eslint-disable-line import/no-extraneous-dependencies
```
Comment (felipe-lee) [inline src/testUtils.jsx:7]: Should this file live in a `tests` folder or something? I feel like it's weird that something that is purely for testing needs this because it's being flagged for using a devDependency. @suzubara ?

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.stories.jsx [hunk 18]
```diff
@@ -0,0 +1,49 @@
...
+  },
+  backupContact: {
+    name: 'Gabriela Sáenz Perez',
+    telephone: '206-555-8989',
+    email: 'gsp@example.com',
+  },
+  editURL: '/moves/review/edit-profile',
+};
+
+export const DefaultState = (argTypes) => <ContactInfoDisplay {...baseProps} onEditClick={argTypes.onEditClick} />;
```
Comment (suzubara) [inline src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.stories.jsx:45]: I think `onEditClick` can be deleted now?

**Review by suzubara — APPROVED** [review]
looks great!! thanks for setting up the DL styles 🎉

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.stories.jsx (hunk 18 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.stories.jsx:45]: Oh, yup, missed that. I'll take that out.

File: src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx (hunk 10 above)
Comment (felipe-lee) [reply src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.jsx]: Funnily enough, changing to use `Link` instead of `Button` made the border much more reasonably sized when clicked
Screen Shot 2021-04-09 at 17 45 14

**Comment by cosburn** [comment]
Everything looks good but one issue with spacing above the "backup contact" header. Attached a screenshot of the design with a spacer element, if that's helpful. I think it looks like about 4 units of spacing. 
Screen Shot 2021-04-12 at 10 38 02 AM

**Comment by felipe-lee** [comment]
> Everything looks good but one issue with spacing above the "backup contact" header. Attached a screenshot of the design with a spacer element, if that's helpful. I think it looks like about 4 units of spacing.
> Screen Shot 2021-04-12 at 10 38 02 AM

Sounds good. Just pushed a change to add more spacing above the header. Hopefully this happo run also produces fewer false changes.

**Review by cosburn — APPROVED** [review]
